app/renderer/lib/client-frameworks/framework.js

```diff
@@ -13,6 +15,23 @@ export default class Framework {
     this.lastAssignedVar = null;
   }
 
+  static getTapCoordinatesFromPointerActions (pointerActions) {
```

Comment:
`static` can be removed. Then, the caller like in java.js can call this method as `this.getTapCoordinatesFromPointerActions` instead of `Framework.getTapCoordinatesFromPointerActions`
```suggestion
  getTapCoordinatesFromPointerActions (pointerActions) {
```

(The below is also the same)

Reply:
Ah right. Done.